README.md

```diff
@@ -199,19 +188,12 @@ kubectl apply -f wavefront-operator-dir/wavefront-operator.yaml
 
 # Removal
 
-###### Helm
+To remove the Wavefront Integration from your environment, use the following helm command.
 
 ```
 helm uninstall wavefront-v2beta
 ```
 
-###### Manual
```

Comment:
I am not sure whether the Custom Resource is cascade deleted on CRD delete by the helm command. Somthing to verify.

Reply:
Yes, on the clean delete. Yes, also, on the cascade verify

Reply:
I fixed the helm uninstall command, it was missing a -n wavefront.  And verified that it correctly does a cascade delete on CRD, etc.  I also add a ` kubectl delete namespace wavefront` command to clean up the namespace